openapi.yaml

```diff
@@ -960,3 +997,109 @@ components:
         approvingTeamDetails:
           type: string
           example: "DevRel"
+    SchemaRequest:
+      title: SchemaRequest
+      required:
+        - schemafull
+      type: object
+      properties:
+        req_no:
+          title: req_no
+          type: integer
+          format: int32
+        topicname:
+          title: Topic name
+          description: Kafka Topic name
+          type: string
+          example: "testtopic"
+        environment:
+          title: environment
+          description: Id of the environment
+          type: string
+          example: "3"
+        environmentName:
+          title: environmentName
+          description: Name of the environment
+          type: string
+          example: "DEV"
+        schemaversion:
+          title: schemaversion
+          description: SchemaRequest version
+          type: string
+          example: "1.0"
+        teamname:
+          title: Team name
+          description: Topic owner team name
+          type: string
+          example: "Infra"
+        teamId:
+          title: Team ID
+          description: Team identifier
+          type: integer
+          format: int32
+          example: 1010
+        appname:
+          title: App name
+          example: "App"
+          type: string
+        schemafull:
+          title: schemafull
+          maxLength: 2147483647
+          minLength: 8
+          type: string
+          example: "{\n  \"doc\": \"exampleTopic\",\n  \"fields\": [\n    {\n      \"default\": \"123456\",\n       \"doc\": \"test\",\n      \"name\": \"test\",\n      \"namespace\": \"test\",\n      \"type\": \"string\"\n }\n  ],\n  \"name\": \"exampleTopic\",\n  \"namespace\": \"exampleTopic\",\n  \"type\": \"record\"\n}"
+        username:
+          title: username
+          description: Username
+          example: jon.snow@klaw-project.io
+          type: string
+        requesttime:
+          title: Request time
+          type: string
+          format: date-time
+          example: 2018-11-13T20:20:39+00:00
+        requesttimestring:
+          title: Request time string representation
+          type: string
+          example: "28-Dec-2022 14:54:57"
+        topicstatus:
+          title: Topic status
+          type: string
+          example: "created"
+        requesttype:
```

Comment:
I think this might also be an enum?

Reply:
I've that as question for Murali on my list. I could not find anything other then it being defined as a string (and the only example I could find was "Create").

Reply:
Yes, same answer as https://github.com/aiven/klaw/pull/384/files/2711db112e1464b0fb6604f4c405854eddfd7faf#r1058807247

Not sure, only suspicions ^^

Reply:
There is an enum for it, but it is not defined properly. You  may leave it for now.
It is here : https://github.com/aiven/klaw/blob/main/core/src/main/java/io/aiven/klaw/model/enums/TopicRequestTypes.java#L8